server/migrations/versions/2023-09-08_add_external_lookup_key_column_to_issue.py

```diff
@@ -0,0 +1,70 @@
+"""Add external_lookup_key column to Issue
+
+Revision ID: 6748b343a12d
+Revises: 983a13692050
+Create Date: 2023-09-08 09:23:43.834532
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# Polar Custom Imports
+from polar.kit.extensions.sqlalchemy import PostgresUUID
+
+# revision identifiers, used by Alembic.
+revision = "6748b343a12d"
+down_revision = "983a13692050"
+branch_labels: tuple[str] | None = None
+depends_on: tuple[str] | None = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "issues", sa.Column("external_lookup_key", sa.String(), nullable=True)
+    )
+    op.create_unique_constraint(
+        op.f("issues_platform_external_lookup_key_key"),
+        "issues",
+        ["platform", "external_lookup_key"],
+    )
+    op.create_index(
+        op.f("ix_issues_external_lookup_key"),
+        "issues",
+        ["external_lookup_key"],
+        unique=False,
+    )
+    op.add_column(
+        "pull_requests", sa.Column("external_lookup_key", sa.String(), nullable=True)
+    )
```

Comment:
Do we really need to have external_lookup_key on the pull requests table? I don't think we ever look up PRs in this way.

Add `external_lookup_key` to `Issue` instead of `IssueFields` to only add it for issues.

Reply:
Yeah, probably not needed, will remove it 👍